src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -13623,9 +13628,25 @@ void func_80850E84(Player* this, PlayState* play) {
 }
 
 static void (*D_80854AA4[])(PlayState*, Player*, void*) = {
-    NULL,          func_80851008, func_80851030, func_80851094, func_808510B4, func_808510D4, func_808510F4,
-    func_80851114, func_80851134, func_80851154, func_80851174, func_808511D4, func_808511FC, func_80851294,
-    func_80851050, func_80851194, func_808511B4, func_80851248, func_808512E0,
+    NULL,
+    Player_AnimChangeIgnoreZeroSpeedXZ,
+    Player_AnimChangeOnceMorphZeroRootYawSpeed2,
+    Player_AnimChangeOnceMorphAdjZeroRootYawSpeed2,
+    Player_AnimChangeLoopMorphAdjZeroRootYawSpeed2,
+    Player_AnimReplaceNormalPlayOnceAdj2,
+    func_808510F4,
+    Player_AnimReplaceNormalPlayLoopAdj2,
+    func_80851134,
+    Player_AnimPlayOnce2,
+    Player_AnimPlayLoop2,
+    func_808511D4,
+    func_808511FC,
+    func_80851294,
+    Player_AnimChangeFreezeZeroRootYawSpeed,
+    Player_AnimPlayOnceAdj2,
+    Player_AnimPlayLoopAdj2,
```

Comment:
All the wrappers to the base functions that end in 2 eg. `Player_AnimPlayOnce2` all are for player cutscenes, so I wonder if the names should reflect that. Similar to every function in `D_80854AA4`

Reply:
wait so are the "2" wrappers also used in the `D_80854AA4` functions? case thats like a layered/priority action system

Reply:
i mixed it up with another function array, disregard

Reply:
Sorry, I phrased that poorly, was mainly referring to other non-2 functions in `D_80854AA4`, but they're all part of this group of functions, and every function in `D_80854AA4` is unique to `D_80854AA4`. Cdi's docs on `D_80854AA4` for reference:

```c
static void (*csModePlaybackFuncs[])(PlayState*, Player*, void*) = {
    NULL,
    Player_AnimPlaybackType0,
    Player_AnimPlaybackType1,
    Player_AnimPlaybackType2,
    Player_AnimPlaybackType3,
    Player_AnimPlaybackType4,
    Player_AnimPlaybackType5,
    Player_AnimPlaybackType6,
    Player_AnimPlaybackType7,
    Player_AnimPlaybackType8,
    Player_AnimPlaybackType9,
    Player_AnimPlaybackType10,
    Player_AnimPlaybackType11,
    Player_AnimPlaybackType12,
    Player_AnimPlaybackType13,
    Player_AnimPlaybackType14,
    Player_AnimPlaybackType15,
    Player_AnimPlaybackType16,
    Player_AnimPlaybackType17,
};
```

Reply:
Yeah I noticed this peculiar set of functions too

The goal of the PR was to name every wrapper somewhat consistently, so I didn't hold off from naming most of the callbacks in this subsystem, but I'd be fine introducing a (probably temporary) subprefix for it

Reply:
imo its fine to just wait until someone documents the cs system

Reply:
reverted